dist/config.yaml

```diff
@@ -124,6 +124,9 @@ gomod:
 announce:
   twitter:
     message_template: '{{ .ProjectName }} {{ .Tag }} is out! Check it out at {{ .ReleaseURL }}'
+  mastodon:
```

Comment:
where does this come from?

Reply:
I ... have no idea TBH. I just ran ./script/build.sh... I can remove them for now, this is not the purpose of this PR anyway. Maybe @tedim52 knows?